test/config/features/FeatureSpec.scala

```diff
@@ -27,18 +27,84 @@ class FeatureSpec extends PlaySpec with GuiceOneAppPerSuite with BeforeAndAfterE
 
   override def beforeEach(): Unit = {
     super.beforeEach()
-    features.simpleAuth(false)
+
+    features.userResearchBanner(true)
+    features.allowDirectDebits(true)
+    features.staticDateEnabled(true)
+    features.vatCertificateEnabled(true)
+    features.enableVatObligationsService(false)
+    features.useDirectDebitDummyPage(true)
+  }
+
+  "The User Research Banner Feature" should {
```

Comment:
I think just testing a single feature should be enough here as it looks like this is testing the .apply() function multiple times.